app/kuma-cp/cmd/run.go

```diff
@@ -114,9 +116,11 @@ func newRunCmdWithOpts(opts kuma_cmd.RunCmdOpts) *cobra.Command {
 				runLog.Error(err, "unable to set up Global KDS")
 				return err
 			}
-			if err := clusterid.Setup(rt); err != nil {
-				runLog.Error(err, "unable to set up clusterID")
-				return err
+			if rt.Config().Store.CreateDefaultResources {
```

Comment:
I don't get these 2 options can you explain?

Reply:
no longer applies